### PR TITLE
iOS Support

### PR DESCRIPTION
**Pull request by thegoldenmule**
Basic iOS support. Currently disabled Trellis connection.

### CODE REVIEW

File: Assets/Source/Application/Injection/EnkluPlayerModule.cs [hunk 1]
```diff
@@ -212,8 +212,10 @@ public void Load(InjectionBinder binder)
                     }
 
 #if UNITY_IOS || UNITY_ANDROID
-                        binder.Bind<IConnection>().To<WebSocketSharpConnection>().ToSingleton();
-                        binder.Bind<IBridge>().To<WebSocketBridge>().ToSingleton();
+                        //binder.Bind<IConnection>().To<WebSocketSharpConnection>().ToSingleton();
+                        //binder.Bind<IBridge>().To<WebSocketBridge>().ToSingleton();
+                        binder.Bind<IConnection>().To<PassthroughConnection>().ToSingleton();
```
Comment (gdamron) [inline Assets/Source/Application/Injection/EnkluPlayerModule.cs]: Please remove commented out code

File: Assets/Source/Player/AR/Ios/IosCameraUpdater.cs [hunk 2]
```diff
@@ -40,6 +41,7 @@ private void Update ()
             _camera.transform.localRotation = UnityEngine.XR.iOS.UnityARMatrixOps.GetRotation(matrix);
             _camera.projectionMatrix = _interface.GetCameraProjection();
         }
+        */
```
Comment (gdamron) [inline Assets/Source/Player/AR/Ios/IosCameraUpdater.cs]: More commented out code to remove! Also, this means `UNITY_IOS` flag is no longer necessary here

File: Assets/Source/Player/Element/Txns/AppSceneManager.cs [hunk 3]
```diff
@@ -43,8 +50,20 @@ public IAsyncToken<Void> Initialize(string appId, IAppDataLoader appData)
...
-
-                _scenes[sceneId] = _elements.Element(description);
+                var element = _scenes[sceneId] = _elements.Element(description);
+                
+                // attach
+                var unityElement = element as IUnityElement;
+                if (null != unityElement)
+                {
+                    unityElement.GameObject.transform.parent = _ar.Root.Transform;
+
```
Comment (gdamron) [inline Assets/Source/Player/Element/Txns/AppSceneManager.cs]: nitpick: this extra white space is not helpful. but we also don't have any guidelines about this in the style guide

Comment (thegoldenmule) [reply Assets/Source/Player/Element/Txns/AppSceneManager.cs]: Feel free to add to the style guide.

Comment (gdamron) [reply Assets/Source/Player/Element/Txns/AppSceneManager.cs]: Thanks, will do.

**Pull request merged**